rasa/core/channels/facebook.py

```diff
@@ -276,10 +292,11 @@ def from_credentials(cls, credentials: Optional[Dict[Text, Any]]) -> InputChanne
             credentials.get("verify"),
             credentials.get("secret"),
             credentials.get("page-access-token"),
+            credentials.get("delay"),
```

Comment:
What happens if this is not set? Should we set it to `0` by default?

Reply:
Probably to `0` or `None` and completely skip the on/off events?

Reply:
Yes. I would say to skip the on/off events in such a case to map current behaviour.

Reply:
ok, if the value of delay is `0` or `None` then we switch off the typing On/Off feature?

Reply:
yes